tests/e2e/specs/scenarios.js

```diff
@@ -2,70 +2,7 @@
 
 /* https://github.com/angular/protractor/blob/master/docs/getting-started.md */
 
-describe("myApp", function() {
+describe("sn.infiniteScroll", function() {
 
-    describe("search", function() {
-
-        beforeEach(function(){
-            browser.manage().deleteAllCookies();
-            browser.get("http://127.0.0.1:8000/");
-            browser.waitForAngular();
-            browser.driver.sleep(2000);
-        });
-
-        it("should automatically redirect to / when location hash/fragment is empty", function() {
```

Comment:
I think we not have some end to end tests for this library, something like we do for FM Frontend mimicking page scrolls.

We can mock a page a with a list of items from a mocked api.